src/machine/machine_stm32_adc_f1.go

```diff
@@ -24,7 +24,7 @@ func InitADC() {
 	enableAltFuncClock(unsafe.Pointer(stm32.ADC1))
 
 	// enable
-	stm32.ADC1.CR2.SetBits(stm32.ADC_CR2_ADON)
+	stm32.ADC1.CR2.SetBits(stm32.ADC_CR2_ADON | stm32.ADC_CR2_ALIGN)
```

Comment:
ADC can `ALIGN` left as we did in code `<< 4`  by setting flag `stm32.ADC_CR2_ALIGN`.